pkg/apis/core/v1/resource_tags.go

```diff
@@ -72,3 +73,21 @@ func ListTags(ctx context.Context, a api.API, obj types.IdentifiedObject) ([]str
 
 	return r.Tags, nil
 }
+
+type taggerImplementation int
+
+func (ti taggerImplementation) Tag(ctx context.Context, a types.API, obj types.IdentifiedObject, tags ...string) error {
+	return Tag(ctx, a, obj, tags...)
+}
+
+func (ti taggerImplementation) Untag(ctx context.Context, a types.API, obj types.IdentifiedObject, tags ...string) error {
+	return Untag(ctx, a, obj, tags...)
+}
+
+func (ti taggerImplementation) ListTags(ctx context.Context, a types.API, obj types.IdentifiedObject) ([]string, error) {
+	return ListTags(ctx, a, obj)
+}
+
+func init() {
+	helper.TaggerImplementation = taggerImplementation(42)
+}
```

Comment:
maybe we should add some comments here, at least this being a workaround and why it is needed